core/src/test/scala/com/microsoft/azure/synapse/ml/core/test/base/TestBase.scala

```diff
@@ -181,6 +181,7 @@ abstract class TestBase extends AnyFunSuite with BeforeAndAfterEachTestData with
   }
 
   protected override def beforeAll(): Unit = {
+    System.setProperty("log4j1.compatibility", "true")
```

Comment:
Do we intend to keep this or was it for debugging?

Reply:
As logs were routed to stderr in Spark3.4, we were not seeing the logs in the tests and build pipeline. This was required to enable the logs back in our test cases